lib/Service/ContactsIntegration.php

```diff
@@ -106,10 +106,12 @@ public function getMatchingRecipient(string $term): array {
 	 * @return false|null|string
 	 */
 	public function getPhoto(string $email) {
-		$result = $this->contactsManager->search($email, ['EMAIL']);
+		$result = $this->contactsManager->search($email, ['EMAIL'], ['escape_like_param' => false]);
 		if (count($result) > 0) {
-			if (isset($result[0]['PHOTO'])) {
-				return $this->getPhotoUri($result[0]['PHOTO']);
+			foreach ($result as $contact){
+				if (isset($contact['PHOTO']) && !empty($contact['PHOTO'])) {
```

Comment:
Negate the condition and `continue` in the `if`, then continue with the early `return` after the `if` block.